.flake8

```diff
@@ -1,4 +1,5 @@
 [flake8]
-exclude=.tox,utils,inventory
+# TODO: cleanup flake8 issues with utils/test/*
```

Comment:
Cool. I may have this covered. I'm cleaning up all tests to be more pytest-ish.
Got it it to a working state but needs some clean up -- tomorrow, as it is quite late here.

#3571